tests/test_examples.py

```diff
@@ -875,6 +878,113 @@ def test_only_non_eq_optimization():
     np.testing.assert_allclose(obj.compute(*obj.xs(surf)), 1, atol=1e-5)
 
 
+@pytest.mark.regression
+@pytest.mark.solve
+@pytest.mark.slow
+def test_freeb_vacuum():
+    """Test for free boundary vacuum stellarator."""
+    extcur = [4700.0, 1000.0]
+    ext_field = SplineMagneticField.from_mgrid(
+        "tests/inputs/mgrid_test.nc", extcur=extcur
+    )
```

Comment:
Where did you get these coil currents from? This mgrid file is in "raw" mode, but the coil currents given in the file itself are 96 and 124

Reply:
See above

Reply:
Again, how did you choose the numbers 4700 and 1000? They feel arbitrary. If you copied it from an example somewhere, reference that example.

Reply:
See above. They're not published anywhere afaik, but these are the values that were in the vmec input along with the mgrid